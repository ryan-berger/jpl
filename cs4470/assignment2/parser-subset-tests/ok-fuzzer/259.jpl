read image "" to m4h.
show P1x.
show 640301.3
assert H, ">"
show K()
write image f() to ""
return .98
time time assert 81, "y"
let G = 4768
let d = 1
show U9D_
write image 7 to "jE"
