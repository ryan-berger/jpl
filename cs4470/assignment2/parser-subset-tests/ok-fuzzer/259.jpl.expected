(ReadImageCmd "" (VarArgument m4h.))
(ShowCmd (VarExpr P1x.))
(ShowCmd (FloatExpr 640301))
(StmtCmd (AssertStmt (VarExpr H) ">"))
(ShowCmd (CallExpr K))
(WriteImageCmd (CallExpr f) "")
(StmtCmd (ReturnStmt (FloatExpr 0)))
(TimeCmd (TimeCmd (StmtCmd (AssertStmt (IntExpr 81) "y"))))
(StmtCmd (LetStmt (ArgLValue (VarArgument G)) (IntExpr 4768)))
(StmtCmd (LetStmt (ArgLValue (VarArgument d)) (IntExpr 1)))
(ShowCmd (VarExpr U9D_))
(WriteImageCmd (IntExpr 7) "jE")
